src/argilla/labeling/text_classification/rule.py

```diff
@@ -74,6 +74,40 @@ def author(self):
         """Who authored the rule."""
         return self._author
 
+    def convert_to_labeling_rule(self):
+        """Converts the rule to a LabelingRule"""
+        if isinstance(self._label, str):
+            labels = [self._label]
+        else:
+            labels = self._label
+
+        return LabelingRule(
+            query=self.query, labels=labels, author=self.author or "None"
+        )
+
+    def update_label(self, label):
+        """Updates the label"""
+        self._label = label
+
+    def add_to_dataset(self, dataset: str):
+        """Add to rule to the given dataset"""
+        api.active_api().add_dataset_labeling_rules(
+            dataset, rules=[self.convert_to_labeling_rule()]
```

Comment:
Here, we have coupled components between `Rule` and `Api`. We've already had it with the `apply` method. We can go forward as is, but we will assume the technical debt. 

If in the future we allow defining contextualized clients, as the example shows:

```python
with rb.init("<an-url>", "<an-api-key") as client:
   client.log..
   client.load...
```
We need to redesign all this stuff.

Just to keep in mind.